apps/src/calc/expressionNode.js

```diff
@@ -101,40 +101,51 @@ ExpressionNode.prototype.canEvaluate = function (mapping) {
 
 /**
  * Evaluate the expression, returning the result.
+ * @param {object} globalMapping Global mapping of variables and functions
```

Comment:
Are mappings in general all variable names (Strings) -> values (Objects)? could then have the param types be `{Object.<string, *>}`